app/src/componentLibrary/systemMessage/systemMessage.scss

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@mixin font-config($font, $size, $height, $color) {
+  font-family: $font;
+  font-size: $size;
+  line-height: $height;
+  color: $color;
+}
+
+@mixin message-headers($color) {
+  color: $color;
+  text-transform: capitalize;
+  margin-bottom: 8px;
+  font-family: $FONT-ROBOTO-MEDIUM;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+@mixin stripe-shape($color--line1, $color--line2) {
+  height: 8px;
+  background-size: 60.81px 60.81px;
+  border-radius: 8px 8px 0 0;
+  background-image: linear-gradient(
+    45deg,
+    $color--line1 26.74%,
+    $color--line2 26.74%,
+    $color--line2 50%,
+    $color--line1 50%,
+    $color--line1 76.74%,
+    $color--line2 76.74%,
+    $color--line2 100%
+  );
+}
+
+.system-message {
+  min-height: 104px;
+  height: auto;
+  width: 100%;
+  background: $COLOR--bg-000;
+  border-radius: 8px;
+  box-shadow: 0px 1px 3px $COLOR--item-shadow;
```

Comment:
Please extract it to the corresponding scss file into variables. $COLOR--item-shadow will be removed from colors variables file as we are using new approach with extracting box-shadow into own vars
$BOX_SHADOW--item: 0px 1px 3px rgba(55, 67, 98, 0.1);